packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -0,0 +1,29 @@
+import { UserAgentApplication as AzureActiveDirectory } from 'msal'
+
+export type { AzureActiveDirectory }
+import type { AuthClient } from './'
+
+export type AzureActiveDirectoryClient = AzureActiveDirectory
+export interface AzureActiveDirectoryUser {}
+
+export const azureActiveDirectory = (client: AzureActiveDirectoryClient): AuthClient => {
+  return {
+    type: 'azureActiveDirectory',
+    client,
+    restoreAuthState: async () => {},
+    login: async (options?) => {
+      await client.loginPopup(options)
+    },
+    logout: (options?) => client.logout(options),
+    signup: async (options?) => {
+      await client.loginPopup(options)
+    },
+    getToken: async () => {
+      return sessionStorage.getItem('msal.idtoken')
+    },
```

Comment:
```suggestion
    getToken: async () => sessionStorage.getItem('msal.idtoken'),
```

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/5c651a3bb959d5b329a1b99800f3a6a2aa1d5d5d